src/lib/types/vec3-float32.js

```diff
@@ -0,0 +1,3 @@
+import r from 'restructure';
+
+export default new r.Array(r.floatle, 3);
```

Comment:
My thought was that we should have a consistent naming scheme, and ```float3``` would conflict with things like ```float16``` (for a single fixed-point 16-bit float).

Since ```float3``` is a vector, I thought maybe we should move toward ```vec3float32``` as a more consistent name.

If you're cool with that, I can search and replace all ```float3```s to ```vec3float32```s